data/json/furniture.json

```diff
@@ -142,6 +150,7 @@
         "type" : "furniture",
         "id" : "f_bulletin",
         "name": "bulletin board",
+        "description": "Pin some notes for other survivors to read.",
```

Comment:
Is this actually a thing in the game? Doesn't the board just mark the location as a camp?
Also, that camp thing would be worth mentioning.

Reply:
"Other survivors to read" is a hint towards camp